supabase/migrations/20220117142104_account_and_org_tables.sql

```diff
@@ -47,18 +74,13 @@ create table app.organization(
 
 create type app.membership_role as enum ('maintainer', 'creator');
 
-create table app.account_in_organization(
+create table app.members(
     id uuid primary key default uuid_generate_v4(),
-    organization_id uuid not null references app.organization(id),
-    account_id uuid not null references app.account(id),
-    member_role app.membership_role not null,
+    organization_id uuid not null references app.organizations(id),
+    account_id uuid not null references app.accounts(id),
+    role app.membership_role not null,
     created_at timestamp not null default (now() at time zone 'utc'),
     unique (organization_id, account_id),
     -- creator is defined on the organization table
```

Comment:
does this mean that the creator will not appear in the members table?

Reply:
correct

The idea was that it'd be more painful for updates/transfers on the backend (hidden behind RPC so it wouldn't impact the API) but a not null owner on `organization` prevents accidentally abandoning a orgs with no members

on second thought, abandoning an organization is a valid use-case so I'll change it.

---
updated in bd50277 to create a `member` record for the creating account after insert on `organizations`